IdentityCore/src/util/MSIDHelpers.m

```diff
@@ -22,6 +22,8 @@
 // THE SOFTWARE.
 
 #import "MSIDHelpers.h"
+#import "MSIDContants.h"
```

Comment:
just realized we had a typo here.
Constants perhaps

Reply:
fixed